src/db/config.ts

```diff
@@ -1,13 +1,17 @@
 import { createConnection } from "typeorm";
+//import { createUser } from "./entity";
+import * as path from "path";
 
 export async function setupDb() {
   console.log("Connecting to database!");
 
   const connection = await createConnection({
     type: "postgres",
     url: "postgres://onboard:onboard@localhost:5432/onboard",
+    entities: [path.join(__dirname) + "/entity/index.{ts,js}"],
```

Comment:
Aqui tem uma mistura que torna o uso do `path` meio desnecessário. A ideia do `path` é montar caminhos de arquivos sem depender do sistema operacional, então ele usaria, por exemplo, `\` no windows e `/` no MAC ou Linux. Mas aí logo em seguida vem as `/` fixas.

Se for manter o path, melhor fazer o join com todo o caminho:

```suggestion
    entities: [path.join(__dirname, 'entity', 'index.{ts,js}'],
```